### DISCUSSION TITLE
How do you change upower battery icon? Or remove it completely?

### DISCUSSION THREAD

**Discussion by Disr0** [JakeStanger/ironbar#457] [Q&A]
I've already tried changing the gtk theme and specifying the icon with -gtk-icon-theme, none of it have had any effect
If there's no way to change the icon, maybe someone knows the way to disable it completely? display:none and visibility:hidden just doesn't work

**Comment by JakeStanger** [comment]
The icon is loaded from the `icon_theme` you specify in your [ironbar config](https://github.com/JakeStanger/ironbar/wiki/configuration-guide#31-top-level-options). Hiding the icon should also definitely be possible. Feel free to share your CSS and I can have a look at where you're going wrong.
